README.md

```diff
@@ -54,8 +54,22 @@ https://www.digitalocean.com/community/articles/geddy-js-a-no-brainer-mvc-node-j
 
 #### License
 
-Apache License, Version 2
+Copyright 2017. Geddy - Web-app development framework
```

Comment:
All our copyrights are listed as 2112, because of this: https://en.wikipedia.org/wiki/2112_(album)

Also, this means we don't have to keep going around keeping a copyright year number up to date. :)

Oh, and also I guess technically I do own the copyright, so it's not really necessary to change either of those lines.